src/diy/smol-slimes/smol-hardware.md

```diff
@@ -0,0 +1,278 @@
+      <td>Stacked 🥪 SuperMini Build</td>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td style="width: 50%">
+        <img
+          src="../../assets/img/smol_slime_schematic.png"
+          alt="Smol Slime Schematic"
+          loading="lazy"
+        />
+      </td>
+      <td style="width: 50%">
+        <img
+          src="../../assets/img/smol_slime_stacked_schematic.webp"
+          alt="Smol Slime (Stacked 🥪) Schematic"
+          loading="lazy"
+        />
+      </td>
+    </tr>
+    <tr>
+      <td></td>
+      <td>
+        Requires special firmware that provides power from the GPIO pins.
+      </td>
+    </tr>
+    <tr>
+      <td></td>
+      <td>
+        <a href="https://youtu.be/qTmIfa_Asic">YouTube Building Tutorial</a> that covers soldering the IMU, battery, and button to the microcontroller board, as well as assembling the case. 
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## 📡 Receiver Hardware
+
+It is essential to use boards equipped with high-quality antennas to ensure signal integrity and range. Hardware featuring PCB antennas is generally the best option for use as a receiver.
+
+### USB Dongles
+
+These dongles have a fairly optimized PCB antenna. If you have issues with signal integrity, it is recommended to use a USB extension cable.
+
+<table>
+  <thead>
+    <tr>
+      <th>Dongle</th>
+      <th>Description</th>
+      <th>Obtaining</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>eByte Dongle (E104-BT5040U)</td>
+      <td>
+        Cheapest option with a PCB antenna.<br />
+        - <strong>E104-BT5040U</strong> is model that you need. It is fully
+        compatible with the Nordic Semiconductor nRF52840 Dongle.<br />
+        - <strong>E104-BT5040UA</strong> is not compatible. It is only capable
+        of capturing BLE4.2 and BLE5.0 protocol packets.
+      </td>
+      <td>
+        It is available on AliExpress, with free shipping.<br />
+        Also available on Alibaba, where sold by
+        <strong>Chengdu Ebyte Electronic Technology Co., Ltd.</strong>
+      </td>
+    </tr>
+    <tr>
+      <td>Nordic Semiconductor nRF52840 Dongle (PCA10059)</td>
+      <td>Official Nordic development hardware.</td>
+      <td>
+        It is available on <strong>Digikey</strong> or <strong>Mouser</strong>.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Microcontrollers Modified Into USB Dongles
+
+```admonish note
+If you plan to use microcontroller as receiver you can modify it to improve signal strength, replacing the built-in antenna with a 31.2 mm wire. This creates a basic monopole antenna.
+```
+
+<table>
+  <thead>
+    <tr>
+      <th>Board</th>
+      <th>Description</th>
+      <th>Obtaining</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>SuperMini nRF52840</td>
+      <td>It is a clone of the <strong>nice!nano</strong> board.</td>
+      <td>
+        Cheapest option overall.<br />
+        It is available on AliExpress with
```

Comment:
```suggestion
        Available on AliExpress with
```
The main reason for this suggestion is this section started with "It is available" while the next starts with "Available". Consistency is important